src/components/backup_clouds/samba/backup_cloud.vue

```diff
@@ -5,15 +5,40 @@
 			<span class="small">(Modul: {{ $options.name }})</span>
 		</openwb-base-heading>
 		<openwb-base-text-input
-			title="Pfad zur Freigabe"
+			title="Server"
+			subtype="host"
 			required
+			:model-value="configuration.smb_server"
+			@update:model-value="
+				updateConfiguration($event, 'configuration.smb_server')
+			"
+		/>
+		<openwb-base-text-input
+			title="Freigabe"
+			required
+			:model-value="configuration.smb_share"
+			@update:model-value="
+				updateConfiguration($event, 'configuration.smb_share')
+			"
+		/>
+		<openwb-base-text-input
+			title="Unterordner (optional)"
 			:model-value="configuration.smb_path"
+			pattern='([^\\:"\|*?<>]+/)*'
 			@update:model-value="
 				updateConfiguration($event, 'configuration.smb_path')
 			"
-		/>
+			>
+			<template #help>
+				Jeder Unterordner muss mit / enden.<br />
+				Die Zeichen \:"|*?&lt&gt sind verboten!<br />
```

Comment:
Bei den HTML Entities fehlt noch das abschließende Semikolon.
```suggestion
				Die Zeichen \:"|*?&lt;&gt; sind verboten!<br />
```